package.json

```diff
@@ -28,13 +28,15 @@
     "build": "npm-run-all --serial \"build:* {@}\" --",
     "build:packages": "yarn ts-node ./scripts/build.ts",
     "build:ts": "yarn ts-node ./scripts/ts-build.ts",
+    "build-storybook": "build-storybook",
```

Comment:
won't this work by default? do we need to write it explicitly in our package.json?

Reply:
The chromatic script tries to run the storybook 7.x script `storybook dev` by default. We can pass in a custom yarn script though, so we need to explicitly define it like this till we upgrade to SB 7.x